<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="SideBar_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>991</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(37, 41, 54);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="ReportTitle">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>400</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>51</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold SemiConden</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;Uploaded File Detection Report&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="SideBar_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>181</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(80, 91, 113);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="menu_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>130</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold SemiConden</family>
       <pointsize>12</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;
font: 63 12pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="text">
      <string>ㅁ File Path</string>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>130</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;
font: 63 12pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="text">
      <string>ㅁ File Size</string>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>130</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;
font: 63 12pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="text">
      <string>ㅁ SHA256 Hash</string>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>160</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;
font: 63 12pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="text">
      <string>ㅁ Detection Result</string>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="menu_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>200</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:left;
font: 63 15pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="text">
      <string>Contents</string>
     </property>
     <property name="icon">
      <iconset resource="Resources/resources.qrc">
       <normaloff>:/icons/menu.png</normaloff>:/icons/menu.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="VersionInfo_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>60</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>61</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold SemiConden</family>
       <pointsize>13</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 13pt &quot;Bahnschrift SemiBold SemiConden&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;v 1.4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="FilePath">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
font: 63 13pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FileSize">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
font: 63 13pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Hash">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>741</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
font: 63 13pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold SemiConden</family>
      <pointsize>13</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:left;
font: 63 13pt &quot;Bahnschrift SemiBold SemiConden&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveCSV">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>340</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold SemiConden</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(67, 196, 133);
	border:none;
	border-radius:15;
}

QPushButton:hover{
	background-color: rgb(59, 173, 116);
}

QPushButton:clicked{
	background-color: rgb(51, 150, 101);
}</string>
    </property>
    <property name="text">
     <string>Save as .xlsx</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
